src/CharterManager.sol

```diff
@@ -214,14 +214,21 @@ contract CharterManagerImp {
         ManagedGemJoinLike(gemJoin).join(getOrCreateProxy(usr), amt);
     }
 
-    function exit(address gemJoin, address u, address usr, uint256 amt) external allowed(u) {
+    function exit(address gemJoin, address usr, uint256 amt) external {
         require(VatLike(vat).wards(gemJoin) == 1, "CharterManager/gem-join-not-authorized");
 
-        address urp = proxy[u];
+        address urp = proxy[msg.sender];
         require(urp != address(0), "CharterManager/non-existing-urp");
         ManagedGemJoinLike(gemJoin).exit(urp, usr, amt);
     }
 
+    function move(address u, address dst, uint256 rad) external allowed(u) {
+        address urp = proxy[u];
+        require(urp != address(0), "CharterManager/non-existing-urp");
+
+        VatLike(vat).move(urp, dst, rad);
+    }
```

Comment:
Why do we need this function? Apart from accidentally sending dai to your proxy it seems like this won't be used anywhere.

Reply:
Nevermind read your comment above.